install.bat

```diff
@@ -0,0 +1,79 @@
+#Cloning the git repository
+#git clone https://github.com/systers/mentorship-backend.git
+
+#This method confirms whether user has entered an input or not
+valid_input(){
+message="$1"
+config_val="$2"
+
+while [ true ]
+do
+if [ -z $config_val ]
+then
+    echo "Error: No input entered"
+    echo -n "$message"
+    read "config_val"
+    continue
+else
+    break
+fi
+done
+}
+
+echo "This project requires Python3. Make sure to install the latest version."
+
+#Installing virtualenv
+pip3 --install user virtualenv
+
+#Creating the virtual environment
+virtualenv env --python=python3
+
+#Activating virtual environment
+source ./env/Scripts/activate
```

Comment:
I'm not sure if this works this way... 

`source` is a UNIX command. The `env/Scripts/activate` is for Windows